### DISCUSSION TITLE
search box?

### DISCUSSION THREAD

**Discussion by ThatWindowsFan** [valinet/ExplorerPatcher#593] [Ideas]
I whould like a fully functional search box for ExplorerPatcher

**Comment by valinet** [comment]
Try https://github.com/stnkl/EverythingToolbar

Microsoft destroyed theirs; it's not available in Windows 11's `explorer`. Granted, it is fairly useless if you ask me. There is a search button which takes way less space and does the same thing as the search box did. Indeed, it can't show ads, something they tried at some early point in Windows 10's lifetime.
